translator/conventions/opentelemetry.go

```diff
@@ -15,18 +15,22 @@
 package conventions
 
 // OpenTelemetry Semantic Convention values for Resource attribute names.
-// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/README.md
 const (
 	AttributeServiceName      = "service.name"
 	AttributeServiceNamespace = "service.namespace"
 	AttributeServiceInstance  = "service.instance.id"
 	AttributeServiceVersion   = "service.version"
-	AttributeLibraryName      = "library.name"
-	AttributeLibraryLanguage  = "library.language"
-	AttributeLibraryVersion   = "library.version"
+	AttributeLibraryName      = "telemetry.sdk.name"
+	AttributeLibraryLanguage  = "telemetry.sdk.language"
+	AttributeLibraryVersion   = "telemetry.sdk.version"
```

Comment:
Please rename to include TelemetrySDK` in the name. I believe these are not used yet anywhere in the contrib so should be safe to rename.

Reply:
Fixed